nservicebus/azure-service-bus/transaction-support.md

```diff
@@ -7,37 +7,31 @@ tags:
 - Transactions
 ---
 
+## Transaction Support
 
-## Transactions and delivery guarantees
+Azure Service Bus Transport supports `SendAtomicWithReceive`, `ReceiveOnly` and `None` transaction mode levels.
 
-NServiceBus Azure Service Bus transport relies on the underlying Azure Service Bus library which requires the use of the `Serializable` isolation level (the most restrictive isolation level that does not permit `dirty reads`, `phantom reads` and `non repeatable reads`; will block any reader until the writer is committed. For more information refer to [Transaction Isolation Levels Explained in Details](http://dotnetspeak.com/2013/04/transaction-isolation-levels-explained-in-details) article.
+### Sends atomic with Receive
 
-NServiceBus Azure Service Bus transport configuration is hard-coded to `Serializable` isolation level. Users can't override it.
+Note: `SendAtomicWithReceive` level is only supported as from version 7. It requires destination and receive queues to be in the same namespace.
```

Comment:
Since we used different names for transaction levels, I think it'd make sense to add here "X is equivalent of `NativeTransactions` in Versions 5 and below" or something along those lines.